README.md

```diff
@@ -1,5 +1,6 @@
 # bulbs-public-ads-manager
-Ads manager for public side of sites. Fills in ad slots.
+Ads manager for public side of sites. Fills in ad slots. 
+See a [simple visual overview](https://docs.google.com/drawings/d/1zwLspXOvd5nVZUH3F_UpoDxqp5ou72XzfyA2QVMlAg0) of the programmatic ad process.
```

Comment:
nice picture!

Reply:
Thanks. Made it initially just to help myself wrap my head around how ads work, but hopefully it's useful to other people coming in cold to the world of GPT, DSPs, header bidding, etc.

On a related note, I didn't realize how obscenely good [Google Drawings](https://en.wikipedia.org/wiki/Google_Drawings) is. I definitely reccomend it if you ever need to do flow charts or things of that ilk.